src/pages/api/user/profile.ts

```diff
@@ -34,9 +34,16 @@ const updateMyProfile: Handler = async (req, res) => {
     if (!checkUsername(req.body?.nick)) {
       throw new Error('Bad username')
     }
-    // validate that URL is properly formatted
-    const website = checkWebsiteUrl(req.body?.website)
-    if (website == null) { throw new Error('Bad website URL') }
+
+    // validate that URL is properly formatted if one is provided
+    let website = req.body?.website
+    if (website !== null && website !== "") {
```

Comment:
I think issue might be here.  When website is `undefined`  website !== null will evaluate to true.  

`website != null` is a better test.

Reply:
Ahh okay. Will wait until getting to my laptop to check it over and make another commit then.